### ISSUE TITLE
Button to add plugin is blocked by multimedia frame

### ISSUE THREAD

**Issue by anbestCL** [serlo/backlog#160]
In the following image, the plus button in the middle cannot be clicked. Resizing the multimedia frame makes it possible to click the button.

![grafik](https://github.com/serlo/backlog/assets/43368821/e70f029e-967e-4e22-a844-9a5664609b13)

(Moved from edtr-io: https://github.com/edtr-io/edtr-io/issues/411)

**Comment by elbotho** [comment]
closing in favour of https://github.com/serlo/backlog/issues/237
